File: scala/src/test/scala/TwoSumTest.scala
import org.scalatest.FunSuite

import leetcode.TwoSum

class TwoSumTest extends FunSuite {
  test("TwoSum") {
    assert(TwoSum.twoSum(Array(2, 7, 11, 15), 9) === Array(0, 1))
  }
}

File: csharp/UnitTest/TwoSumTest.cs
using Leetcode;
using Xunit;

namespace LeetcodeTest
{
    public class TwoSumTest
    {
        [Theory]
        [InlineData(new int[] { 2, 7, 11, 15 }, 9, new int[] { 0, 1 })]
        public void TestMethod(int[] nums, int target, int[] output)
        {
            Assert.Equal(output, new TwoSumSolution().Solution(nums, target));
        }
    }
}
